employee.rb

```diff
@@ -0,0 +1,21 @@
+  require("./assignment3.rb")
```

Comment:
I don't think you need to require assignment3.rb file here.

Reply:
Yes, removed require